### ISSUE TITLE
REGEXP finds data but displays it without trailing zeroes

### ISSUE THREAD

**Issue by 71GA** [dbeaver/dbeaver#1716]
I am using Dbeaver client for MariaDB and when I use `REGEXP '\.[0-9]'` on a table column, Dbeaver will properly find values (for example `2.50`) but when it will display them it will hide the trailing zeroes (for example `2.5` instead of `2.50`).

So an inexperienced user like me gets confused easily! At first I thought `REGEXP '\.[0-9]'` ain't working and I lost quite some time to figure this one out.

So does anyone know how to enable trailing zeroes in Dbeaver?

Source: https://superuser.com/questions/1213765/dbeaver-mysql-workbench-different-data-formatting-for-trailing-zeroes

**Comment by serge-rider** [comment]
Please post table DDL with INSERT.

Possible workaround is to configure data formatting for numbers (set "Minimum fraction digits to 2). However this looks like MySQL-specific issue which should be handled properly in DBeaver).

**Comment by 71GA** [comment]
CAn you give me an exact command that I need to input?

**Comment by serge-rider** [comment]
DDL is on DDL tab of a table editor.
You can generate INSERT by right clicking on table data->Generate SQL->INSERT.

**Comment by 71GA** [comment]
Ok so I right clicked the table named "products" and followed link **/Generate SQL/INSERT** and I got this query: 

```
INSERT INTO crashcourse.products
(prod_id, vend_id, prod_name, prod_price, prod_desc)
VALUES('', 0, '', 0, '');
```

**Comment by serge-rider** [comment]
What about DDL?

**Comment by 71GA** [comment]
Oh sorry. This is the DDL: 
```
CREATE TABLE `products` (
  `prod_id` char(10) COLLATE utf8mb4_unicode_ci NOT NULL,
  `vend_id` int(11) NOT NULL,
  `prod_name` char(255) COLLATE utf8mb4_unicode_ci NOT NULL,
  `prod_price` decimal(8,2) NOT NULL,
  `prod_desc` text COLLATE utf8mb4_unicode_ci,
  PRIMARY KEY (`prod_id`),
  KEY `fk_products_vendors` (`vend_id`)
) ENGINE=Aria DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci PAGE_CHECKSUM=1;

```

**Comment by serge-rider** [comment]
Sorry for a long delay.
Eventually we'll fix numbers formatting in 4.3.5.
